spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java

```diff
@@ -59,8 +59,8 @@
 public abstract class TestDelete extends SparkRowLevelOperationsTestBase {
 
   public TestDelete(String catalogName, String implementation, Map<String, String> config,
-                    String fileFormat, Boolean vectorized) {
-    super(catalogName, implementation, config, fileFormat, vectorized);
+                    String fileFormat, Boolean vectorized, String distributionMode) {
+    super(catalogName, implementation, config, fileFormat, vectorized, distributionMode);
```

Comment:
So as to minimize the diff in this PR (larger diffs can make it harder to backport these changes if people have forks), would it make sense to add a second constructor that has `distributionMode` defaulted, so that only the tests that need to pass `distributionMode` have their files updated in this PR?

Not sure if the change would be here or `SparkRowLevelOperationsTestBase`, will leave that up to you =)

If only a few places were updated to use the constructor and don't make use of the `distributionMode`, you can ignore / resolve this =)

Reply:
Ah I see only a few small places were updated. You can resolve this at your leisure. PR looked bigger than it really is at first glance 😅